karmagrambot/commands.py

```diff
@@ -39,7 +39,15 @@ def karma(_: Bot, update: Update):
 
     _, *args = text.split()
 
-    username = args[0].lstrip('@') if args else None
+    username = None
+    period = get_period('m')
```

Comment:
We really need a better argument handling solution :(.

Reply:
totally agree :s

Reply:
Maybe creating a custom command handler?